extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanClientProducer.java

```diff
@@ -167,12 +166,31 @@ private ConfigurationBuilder builderFromProperties(Properties properties) {
             builder.marshaller((Marshaller) marshallerInstance);
         }
 
-        // Override serverList property value at runtime if such configuration exists
         if (infinispanClientRuntimeConfig != null) {
-            Optional<String> runtimeServerList = infinispanClientRuntimeConfig.serverList;
-            if (runtimeServerList.isPresent()) {
-                properties.put(ConfigurationProperties.SERVER_LIST, runtimeServerList.get());
-            }
```

Comment:
I have an issue with this change. It doesn't look related and could potentially break things. Is it safe? Is the server list still overridable at runtime?

Reply:
yes, it actually the most frequently overridden property as the Infinispan server will differ between different environments

Reply:
see here https://github.com/quarkusio/quarkus/pull/4502#discussion_r335095046

Reply:
Ah I totally missed it was still there.